OMS.Web/OMS.Web.Services/Commands/IsolateOutageCommand.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace OMS.Web.Services.Commands
+{
+    public class IsolateOutageCommand : OutageLifecycleCommandBase
+    {
+        public IsolateOutageCommand(long gid): base(gid) { }
```

Comment:
Već smo nailazili na probleme zbog nedoumice gid Elementa ili gid Merenja, a sad nam se pojavljuje i OutageId koji je auto increment, pa predlažem da to negde jasno naglasimo kad je šta u pitanju. Mozda ja nisam dobro razumeo ovu klasu, ali mi se cini da ce ovde ici long outageId, pa treba skladno promeniti i na svim ostalim mestima gde pisemo samo gid. Okej mi je u privatnim metodama i na NMS da se kaže samo gid, ali u kontraktima i akcijama da bas stoji measurementId (ili measurementGid), elementId (odnosno measurementGid). Id ispada svakako nije gid, pa bi trebalo samo outageId.